media-libs/libdiscid/libdiscid-0.6.2.recipe

```diff
@@ -4,19 +4,23 @@ disc IDs from audio CDs. It reads a CD's table of contents (TOC) and \
 generates an identifier which can be used to lookup the CD at MusicBrainz. \
 Additionally, it provides a submission URL for adding the disc ID to the \
 database and gathers ISRCs and the MCN (=UPC/EAN) from disc."
-HOMEPAGE="http://musicbrainz.org/doc/libdiscid"
-COPYRIGHT="2012 Andrew Hawkins"
+HOMEPAGE="https://musicbrainz.org/doc/libdiscid"
+COPYRIGHT="2013-2017 Robert Kaye et al"
```

Comment:
As there're many more copyright holders, you could instead use "2006-2013 MetaBrainz Foundation" as the value for `COPYRIGHT` then install the `AUTHORS` file included in the distribution.

Reply:
That's a good idea. Thanks @alaviss for the suggestion!
Hmm, did you really mean "2006-201**3** MetaBrainz Foundation" ?
Or should we use "2006-201**7** MetaBrainz Foundation" ?

@sfanxiang, if you agree, could you please make the following changes? Thanks!
```diff
-COPYRIGHT="2013-2017 Robert Kaye et al"
+COPYRIGHT="2006-2017 MetaBrainz Foundation"
```
(I'm not sure about the year.)

```diff
 INSTALL()
 {
 	make install
+	install -d $docDir
+	install -t $docDir AUTHORS README
 
 	rm $libDir/libdiscid.la
```

Reply:
@fbrosson Sorry, I forgot to link to [reference][0]. [Scanning][1] through the release there aren't any copyright that gone further than 2013. I suggest you fix this one.

[0]: https://github.com/metabrainz/libdiscid/blob/v0.6.2/versioninfo.rc.in#L23
[1]: https://github.com/metabrainz/libdiscid/search?p=1&q=copyright&type=&utf8=%E2%9C%93

Reply:
No problem :)
But since the timestamp of the latest tarball, `libdiscid-0.6.2.tar.gz`, is:
```
Last-Modified: Sun, 29 Jan 2017 14:00:22 GMT
```
I'd say that we can keep 2017. What do you think?

Reply:
Even the ChangeLog has a timestamp in 2017: `2017-01-29 12:43:54 UTC`.
How come would the recent changes not also be Copyright MetaBrainz Foundation ?

But maybe I'm missing something. Would you say that the changes after 2013 should have another copyright?
Or would you say that we should not update our copyright years if upstream forgets to do so in their files? (I don't like this idea, but that's a point of view.)

Reply:
I don't believe we've the rights to bump copyright year without authors permission. But IANAL, so we would need some legal advice from a lawyer.

Reply:
Well, the information we provide in our recipes is not authoritative. It's only informative.
Now, there are 2 possible situations:
* if the "legally" correct year is 2013 and we write 2017, nobody will ever complain.
* if the "legally" correct year is 2017 and we write 2013, then the owners could ask: _"Why didn't you write 2017? You know the archive was released in 2017, so you should assume the authors forgot to update the metadata!"_ And we would have to answer: _"We did not dare to refresh the copyright years in our archive..."_

For these reasons I think we can keep 2017. Moreover, writing 2013 might make potential users think the library is not maintained.

Reply:
@fbrosson @alaviss
IANAL either, but I found this clause [here](https://www.copyright.gov/title17/92chap4.html):
> in the case of compilations or derivative works incorporating previously published material, the year date of first publication of the compilation or derivative work is sufficient.

And IMO, I can't find anything against writing 2017 as the year.

Of course, the actual situations can be much more complex.